charts/stable/teamspeak3/values.yaml

```diff
@@ -31,6 +31,7 @@ service:
   query:
     ports:
```

Comment:
Still service not enabled.

Reply:
Sorry. I shouldnt do such things. When i just woke up. Added now